### PR TITLE
Fixes for next minor release

### PR DESCRIPTION
**Pull request by adolfo-rt**
- Spawner / unspawner fixes.
- Misc doc fixes
- Make soft pos limits open loop

### CODE REVIEW

**Comment by adolfo-rt** [comment]
I had some minor ros_control fixes in the pipeline. Now that @davetcoleman has made a ros_controllers release, we could bump ros_control as well. If there are no objections, I'll merge this PR and release ros_control tomorrow.

File: controller_manager/scripts/spawner [hunk 1]
```diff
@@ -57,20 +57,15 @@ load_controller_service = ""
 switch_controller_service = ""
 unload_controller_service = ""
 
+
 def shutdown():
     global loaded,unload_controller_service,load_controller_service,switch_controller_service
 
-    return #temp
```
Comment (davetcoleman) [inline controller_manager/scripts/spawner:63]: thanks for fixing that. i don't remember why that's there but my bad.

File: joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h [hunk 2]
```diff
@@ -58,20 +60,51 @@ T saturate(const T val, const T min_val, const T max_val)
...
+ *
+ * - Actual velocity values are \e not used because of the above reason, and because some platforms might not expose
+ *   trustworthy velocity measurements, or none at all.
+ *
+ * The downside of the open loop behavior is that velocity limits will not be enforced when recovering from large
+ * position tracking errors. Only the command is guaranteed to comply with the limits specification.
+ *
+ * \note: This handle type is \e stateful, ie. it stores the previous position command to estimate the command
+ * velocity.
+ */
```
Comment (davetcoleman) [inline joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h:91]: beautiful, well done!

**Comment by davetcoleman** [comment]
I did a quick visual review, I can test it tomorrow or you can just go ahead and release it.

**Pull request merged**